README.MD

```diff
@@ -54,23 +54,23 @@ This also enables a really simple pattern for clickable icon:
 Include in your `package.json` using the current head commit after the `#`:
 ``` json
 {
-    "svg-color-loader": "https://github.com/SwiftSoftware-Inc/svg-color-loader.git#e7f8eb6",
+    "svg-color-loader": "https://github.com/SwiftSoftware-Inc/svg-color-loader.git#_HEAD_COMMIT_HASH_",

```

Comment:
Blocker: Please change the URL to `HighGearSoftware` instead of `SwiftSoftware-Inc` to reflect the updated repo name

Reply:
Good catch!